packages/cli/src/public-api/v1/handlers/workflows/workflows.handler.ts

```diff
@@ -120,6 +121,10 @@ export = {
 				return res.status(404).json({ message: 'Not Found' });
 			}
 
+			if (excludePinnedData) {
```

Comment:
we don't have to fix this here, but maybe we should create a tech-debt ticket to remove this at the query level, instead of using `delete` on the returned data.